hnn_core/network.py

```diff
@@ -417,6 +849,56 @@ def plot_cells(self, ax=None, show=True):
         return plot_cells(net=self, ax=ax, show=show)
 
 
+class NetworkDrive(dict):
```

Comment:
let's make this private. Not sure yet if we want users to be messing around with this

Reply:
On second thought, -1: just realised that's where the documentation for the different keys lies. Should we not leave it visible for transparency?